README.rst

```diff
@@ -189,6 +189,64 @@ If you want to undo that operation and effectively *remove* everything the playb
         deployment/playbooks/manage-cloudigrade.yml
 
 
+Azure Account Setup
+-------------------
+
+Next you'll need an Azure account. You can sign up for one, or login `here <https://portal.azure.com/>`_. You will need to get the tenant id, client id, subscription id, and the client secret.
+
+The tenant id is your Azure directory id, the subscription id is as it says. To get the client id and secret follow these steps:
+
+#. Log into the `Azure Portal <https://portal.azure.com/>`_
+#. Navigate to the Azure Active Directory Blade.
+#. In the left column, under `Manage`, select `App Registrations`.
+#. Select `New Registration`
+#. Name your app e.g. `cloudigrade-dev-kb`
+#. Click Register. You should now be taken to your new App Registration.
+#. Note your `Application (client) ID` on the Overview page, this is your `Client ID`.
+#. In the left column, under `Manage`, select `Certificates & secrets`.
+#. Select `New client secret`
+#. Add a helpful description and expiration date.
+#. Click Add. Your `Client Secret` is under the `Value` column.
+
+After you've acquired those values, set the environment variables:
+
+- ``AZURE_CLIENT_ID="your client id from above"``
+- ``AZURE_CLIENT_SECRET="your client secret from above"``
+- ``AZURE_SUBSCRIPTION_ID="your azure subscription id"``
+- ``AZURE_TENANT_ID="your azure directory id"``
+
+Finally, before your deployment is able to talk to Azure, you'll need to create a role with all the necessary permissions.
+
+#. Log into the `Azure Portal <https://portal.azure.com/>`_
+#. Navigate to the Azure Subscription that you'll be using.
+#. In the left column, select `Access control (IAM)`.
+#. Select `Add -> Custom Role`
+#. Name the role, click `Next`.
+#. Select the `JSON` tab.
+#. Paste the following JSON block replacing the permissions array only:
+
+.. code-block:: json
+
+        "permissions": [
+            {
+                "actions": [
+                    "Microsoft.Compute/skus/read"
+                ],
+                "notActions": [],
+                "dataActions": [],
+                "notDataActions": []
+            }
+        ]
+
+#. Click `Review + create` -> `Create`.
```

Comment:
This is a pretty trivial point, but I figured I'd mention in case you didn't notice. If you intended for this number list to continue counting where it stopped before the code block (instead of resetting back to 1), *I think* the code block needs to be indented to make the parser think it's part of the previous `<li>`'s content.

I was looking at the formatted version here: https://github.com/cloudigrade/cloudigrade/blob/5b883be4cddc6f1a510bea8a1fe4738085b40a43/README.rst#azure-account-setup

It's really not a big deal, and I wasn't even sure about the intent. Feel free to disregard and resolve this comment if you like. 🙂